### PR TITLE
Offer a choose to user which allow test in config

### PR DESCRIPTION
**Pull request by supdrewin**
Well, I think that a simple capture.py could be used for testing ir-emitter flexible. At the same time, people could also test their camera by their own way more like to. At line 44 of the auto-config.py, I set the default camera to video2 and default test time to 5 sec. These values can be changed at any time. The patch has been tested by myself.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+auto/enable-ir-emitter
```
Comment (supdrewin) [inline .gitignore:1]: Ignore some build file.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+auto/enable-ir-emitter.service
```
Comment (supdrewin) [inline .gitignore:2]: Add to ignore some build file to avoid pull argument.

**Review by supdrewin — COMMENTED** [review]
A build folder should be considered.🤔

**Comment by EmixamPP** [comment]
Yeah it's nice ! I will look at that saterday for merge. 

But don't put too much energy into this script. In a few weeks, something more advanced will appear. I will of course introduce what you did earlier.

**Comment by EmixamPP** [comment]
It's merge :smiley:

**Pull request merged**